### ISSUE TITLE
explore adding patient and specimin id

### ISSUE THREAD

**Issue by scottcain** [cancogen-virus-seq/portal#197]
more data

**Comment by b-f-chan** [comment]
@scottcain - Need a few more details :) - Do you mean adding these fields to the UI?  The metadata download?  Both?  Or maybe other :)

**Comment by scottcain** [comment]
Yeah, basically, one of the PIs suggested that this might be something we may need "the near future" but was otherwise pretty vague. My guess at the moment is that this isn't going to go anywhere.
